scripts/project.js

```diff
@@ -26,6 +26,15 @@
     });
   };
 
+  Project.projectDescWordCount = function(){
+    return Project.all.map(function(project){
+      return project.projectDescription.split(/\b\S+\b/g).length
+    })
+    .reduce(function(prevTotal, curr){
+      return prevTotal + curr;
+    })
```

Comment:
Should probably make prevTotal and curr more semantic. totalCount and indivCount, maybe? Also, is chaining the best choice here? It seems like the individual wordcounts are something I might want to access elsewhere. Maybe this should be split into two modules.